src/pages/index.js

```diff
@@ -163,7 +163,7 @@ export default function IndexPage({ pkgVersion }) {
             />
           </div>
         </div>
-        <div className="flex justify-between items-start shadow-banner px-4 lg:flex-row sm:flex-col sm:px-6 lg:px-6 bg-white">
+        <div className="flex justify-between items-start shadow-banner px-4 lg:flex-row sm:flex-col sm:px-6 lg:px-6 bg-white flex-col sm:flex-row">
```

Comment:
Looks like you have two conflicting styles here. There are two setting the flex direction sm:flex-col and on the end now sm:flex-row